qiita_db/artifact.py

```diff
@@ -618,13 +618,17 @@ def delete(cls, artifact_id):
                      WHERE artifact_id IN %s"""
             qdb.sql_connection.TRN.add(sql, [all_ids])
 
+            # do not move these files back to upload folder.
```

Comment:
Only one question: why add a parameter to `move_filepaths_to_upload_folder` vs. adding it directly to it? I mean there is already a `if x['fp_type'] in ('html_summary', 'html_summary_dir'):` where the `do_not_move` list could also be checked. What do you think? My concern is that now we have 2 places to define this.

Reply:
I'm for that. It is true we are already hard-coding one list, and the hardcode seems reasonable.